README.md

```diff
@@ -722,6 +722,16 @@ Reverses socket connections from the device (remote) to the ADB server host (loc
 * Returns: `Promise`
 * Resolves with: `true`
 
+#### client.root(serial[, callback])
+
+Puts the device into root mode which may be needed by certain shell commands.  A remount is generally required after a successful root call
```

Comment:
Could you add a final period, and perhaps something like: **Note that this will only work if your device supports this feature. Production devices almost never do.**

I just know that someone will ask eventually why it's not working...